src/Tests.Microsoft.Security.Utilities.Core/RE2RegexEngine.cs

```diff
@@ -36,6 +33,28 @@ public IEnumerable<UniversalMatch> Matches(string input, string pattern, RegexOp
                     };
                 }
             }
+            else
+            {
+                if (Regex2.Matches(pattern, input, out List<Dictionary<string, FlexMatch>> matches, 256L * 1024L * 1024L))
```

Comment:
It would be nice to use a named argument or constant here. I was confused thinking this might be max input or match length, but it's max memory.

Reply:
Hmm, seems there was a race in my review and this getting fixed.